### ISSUE TITLE
[Image needs adding] "Unionisation" chapter

### ISSUE THREAD

**Issue by Arielle-Bennett** [the-turing-way/the-turing-way#3344]
### Summary

The chapter: https://the-turing-way.netlify.app/ethical-research/activism/activism-unionisation does not have an image included in the text. One was created in a previous Book Dash but it now needs to be embedded into the guide and alt text written. 

This is linked to issue #3331 which has more detailed instructions!

### What needs to be done?

- [ ] Find the image from the Zenodo library (https://zenodo.org/records/8169292)
- [ ] Write the alt text for the image
- [ ] Open a PR to add the image to the chapter 
- [ ] PR reviewed and merged 

### Who can help?

* Anyone, this is a good first issue 

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

**Comment by Arielle-Bennett** [comment]
For anyone eyeing up possibilities, I'm going to add the image for this one 😄
